### ISSUE TITLE
Rule L060 not applied on dialect 'tsql'

### ISSUE THREAD

**Issue by delta32000** [sqlfluff/sqlfluff#4107]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

With :
```Python
sql = """SELECT ISNULL(MY_VALUE,0) FROM MY_TABLE"""
```
the fix([...], dialect = 'tsql') method does not apply the Rule_L060.
It's applied if I change to IFNULL, but in MSSQL world they chosed to use the keyword ISNULL :/

### Expected Behaviour

the fix() method should return : 
```SQL
SELECT COALESCE(MY_VALUE, 0) FROM MY_TABLE;
```

### Observed Behaviour

It returns : 
```SQL
SELECT ISNULL(MY_VALUE, 0) FROM MY_TABLE;
```

### How to reproduce

```Python
sql = """SELECT ISNULL(MY_VALUE,0) FROM MY_TABLE"""
print(sqlfluff.fix(sql, dialect = 'tsql'))
```

### Dialect

tsql

### Version

1.4.0

### Configuration

.sqlfluff

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by fmms** [comment]
I am not sure how this is in the other SQL dialect. However, in tsql, there is a difference between coalesce and isnull.

see https://learn.microsoft.com/en-us/sql/t-sql/language-elements/coalesce-transact-sql?view=sql-server-ver16 for the details

**Comment by peterbud** [comment]
Note: The rule now called `CV02`

**Comment by peterbud** [comment]
I agree with @fmms, `ISNULL` and `COALESCE` is different enough, it would be dangerous to just replace `ISNULL` blindly. Closing the issue
